packages/core/redux/wallets_btc/actions.js

```diff
@@ -0,0 +1,309 @@
+/**
+ * Copyright 2017–2018, LaborX PTY
+ * Licensed under the AGPL Version 3 license.
+ */
+
+import {
+  BLOCKCHAIN_BITCOIN,
+  BLOCKCHAIN_BITCOIN_CASH,
+  BLOCKCHAIN_BITCOIN_GOLD,
+  BLOCKCHAIN_ETHEREUM,
+  BLOCKCHAIN_LITECOIN,
+  BLOCKCHAIN_NEM,
+  BLOCKCHAIN_WAVES,
+  WALLET_HD_PATH,
+} from '@chronobank/login/network/constants'
+import WalletModel from '../../models/wallet/WalletModel'
+import { subscribeOnTokens } from '../tokens/actions'
+import { formatBalances, getWalletBalances } from '../tokens/utils'
+import TokenModel from '../../models/tokens/TokenModel'
+import tokenService from '../../services/TokenService'
+import Amount from '../../models/Amount'
+import { getAccount } from '../session/selectors'
+import ethereumDAO from '../../dao/EthereumDAO'
+import { getMainEthWallet, getWallets } from './selectors/models'
+import { notifyError } from '../notifier/actions'
+import { DUCK_SESSION } from '../session/constants'
+import { AllowanceCollection, SignerMemoryModel } from '../../models'
+import { executeTransaction } from '../ethereum/actions'
+import {
+  WALLETS_SET,
+  WALLETS_SET_NAME,
+  WALLETS_UPDATE_BALANCE,
+  WALLETS_UPDATE_WALLET,
+} from './constants'
+import { getPersistAccount } from '../persistAccount/selectors'
+
+const isOwner = (wallet, account) => {
+  return wallet.owners.includes(account)
+}
+
+export const get2FAEncodedKey = (callback) => () => {
+  return ethereumProvider.get2FAEncodedKey(callback)
+}
+
+export const setWalletName = (walletId, name) => (dispatch) => dispatch({ type: WALLETS_SET_NAME, walletId, name })
```

Comment:
change to this: 
`
export const setWalletName = (walletId, name) => ({ type: WALLETS_SET_NAME, walletId, name })
`